bodhi/tests/server/consumers/test_greenwave.py

```diff
@@ -0,0 +1,108 @@
+                'msg_id': '2019-09aceb45-3360-4f83-9975-32ae6439c41c',
+                'crypto': 'x509',
+                'topic': 'org.fedoraproject.prod.greenwave.decision.update',
+                'signature': '100% real please trust me',
+                'msg': {
+                    'subject_type': 'koji_build', 
+                    'policies_satisfied': True, 
+                    'decision_context': 'bodhi_update_push_testing', 
+                    'satisfied_requirements': [
+                        {
+                            'testcase': 'dist.abicheck', 
+                            'type': 'test-result-passed', 
+                            'result_id': 27968725
+                        },
+                        {
+                            'testcase': 'dist.rpmdeplint', 
+                            'type': 'test-result-passed', 
+                            'result_id': 28670567
+                        }
+                    ],
+                    'product_version': 'fedora-30', 
+                    'applicable_policies': [
+                        'taskotron_release_critical_tasks_for_testing'
+                    ],
+                    'unsatisfied_requirements': [], 
+                    'subject_identifier': 'bodhi-3.13.3-1.fc30', 
+                    'subject': [
+                        {
+                            'item': 'bodhi-3.13.3-1.fc30', 
+                            'type': 'koji_build'
+                        }
+                    ],
+                    'summary': 'All required tests passed', 
+                    'previous': {
+                        'satisfied_requirements': [
+                            {
+                                'testcase': 'dist.abicheck', 
+                                'type': 'test-result-passed', 
+                                'result_id': 27968725
+                            },
+                            {
+                                'testcase': 'dist.rpmdeplint', 
+                                'type': 'test-result-passed', 
+                                'result_id': 28666088
+                            }
+                        ],
+                        'summary': 'All required tests passed', 
+                        'policies_satisfied': True, 
+                        'unsatisfied_requirements': [], 
+                        'applicable_policies': [
+                        'taskotron_release_critical_tasks_for_testing'
+                        ],
+                    }
+                }
+            },
+        )
+        self.handler = greenwave.GreenwaveHandler()
+
+    @mock.patch('bodhi.server.consumers.greenwave.Build')
+    def test_update_test_gating_status(self, mock_build_model):
+        """Assert that messages marking the build as signed updates the database"""
+        build = mock_build_model.get.return_value
+        update = models.Build.get("bodhi-3.13.3-1.fc30").update
+        update.test_gating_status = ""
```

Comment:
This is an enum:

https://github.com/fedora-infra/bodhi/blob/3.14.0/bodhi/server/models.py#L638-L658

Reply:
```suggestion
        update.test_gating_status = models.TestGatingStatus.waiting
```